.docker/Makefile

```diff
@@ -0,0 +1,13 @@
+GIT_TAG ?= master
+TAG ?= latest
+
+all: image-tier1 image-tier2 image-tier3
```

Comment:
Make all these targets `.PHONY` otherwise very bad things happen when you have a file called `all` or `image-tier1`.